sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/AzureTable.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.data.tables;
```

Comment:
You're missing the MIT header. If you look at some previous *.java files, they'll have two lines at the header for every file.